blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -20,6 +20,8 @@ export default class PipelinePage extends Component {
             return null; // Loading...
         }
 
+        const baseUrl = removeLastUrlSegment(this.context.location.pathname);
```

Comment:
You're right, in this case we can just use `/organizations/${organization}/${pipelineName}/ as the baseUrl. I wonder if instead of hardcoding the URL paths in various views if there would be any benefit to a utility that would create the URLs for us? From a maintenance standpoint, it might be easier if they were in one place.